packages/shared/src/archive-storybook/index.ts

```diff
@@ -28,8 +28,13 @@ export function buildArchiveStorybook(
 }
 
 function binPath() {
-  // eslint-disable-next-line global-require
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
   const packageJson = require('storybook/package.json');
+  const parsedVersion = parseInt(packageJson.version[0], 10);
```

Comment:
How does the `packageJson.version[0]` work? Is that field aleady split into segments from the `require`?

Reply:
@tevanoff It's not, it's a version number string -- It's just grabbing the first character (major version number).

Reply:
Ah, I see! This will be `1` when SB 10 comes out, will that be a problem? Should we split on `.` to ensure it's not?

Reply:
Woah good catch. Will fix!